packages/jspsych/src/modules/plugins.ts

```diff
@@ -138,7 +144,9 @@ export type UniversalPluginParameters = InferredParameters<typeof universalPlugi
 
 export interface PluginInfo {
   name: string;
+  version?: string;
```

Comment:
The changes to these files will be part of #3326 instead, right? Just thinking about whether we need a changeset for `jspsych`

Reply:
This is correct, do not need changeset for `jsPsych`. Corrected changes by reverting these lines.